--- lilo-config/CMakeLists.txt	2008-01-29 10:18:52.000000000 +0100
+++ lilo-config/CMakeLists.txt-new	2008-07-22 15:03:41.122766250 +0200
@@ -2,7 +2,7 @@
 
 add_definitions (-DQT3_SUPPORT -DQT3_SUPPORT_WARNINGS)
 
-FIND_PROGRAM(LILO_EXECUTABLE NAMES lilo LILO  PATHS /sbin /usr/sbin /usr/local/sbin /opt/lilo  )
+SET(LILO_EXECUTABLE /sbin/lilo)
 
 if(LILO_EXECUTABLE)
 	ADD_DEFINITIONS(-DUSE_KDE ) # added manually
