--- kdeadmin-4.9.4/system-config-printer-kde/system-config-printer-kde.py.orig	2012-11-30 01:27:53.000000000 +0100
+++ kdeadmin-4.9.4/system-config-printer-kde/system-config-printer-kde.py	2012-12-25 22:59:48.067167781 +0100
@@ -35,8 +35,7 @@
 
 #load modules from system-config-printer-common (debug, smburi), change path here if you have it installed elsewhere
 SYSTEM_CONFIG_PRINTER_DIR = "/usr/share/system-config-printer"
-if os.path.exists(SYSTEM_CONFIG_PRINTER_DIR + "/debug.py"):
-    sys.path.append(SYSTEM_CONFIG_PRINTER_DIR)
+sys.path.append(SYSTEM_CONFIG_PRINTER_DIR)
 
 from PyQt4.QtCore import *
 from PyQt4.QtGui import *
